index.php

```diff
@@ -0,0 +1,127 @@
+            <ul>No</ul>
+
+        </div>
+
+        <div class="plants">
+            <img class="elephant_ear" src="images/elephant-ear-plant.jpg" alt="beautiful elephant ear plant with a blue background">
+            <p>Elephant Ear Plant</p>
+            <h2>BOTANICAL NAME:</h2>
+            <ul>Alocasia zebrina</ul>
+            <h2>NICKNAME:</h2>
+            <ul>Elephant ear plant</ul>
+            <h2>PLANT TYPE:</h2>
+            <ul>Perennial, indoor</ul>
+            <h2>PLANT HEIGHT:</h2>
+            <ul>55.5 cm</ul>
+            <h2>POISENOUS:</h2>
+            <ul>Yes</ul>
+        </div>
+
+        <div class="plants">
+            <img class="spider_plant" src="images/curly-spider-plant.jpg" alt="curly spider plant in a grey pot">
+            <p>Curly Spider Plant</p>
+            <h2>BOTANICAL NAME:</h2>
+            <ul>Chlorophytum comosum</ul>
+            <h2>NICKNAME:</h2>
+            <ul>Curly spider plant; Variegated spider plant; Airplane plant</ul>
+            <h2>PLANT TYPE:</h2>
+            <ul>Perennial indoor</ul>
+            <h2>PLANT HEIGHT:</h2>
+            <ul>30 cm</ul>
+            <h2>POISENOUS:</h2>
+            <ul>No</ul>
+
+        </div>
+
+        <div class="plants">
+            <img class="ivy" src="images/ivy.jpg" alt="english ivy in a grey pot with white background">
+            <p>English Ivy Plant</p>
+            <h2>BOTANICAL NAME:</h2>
+            <ul>Hedera helix</ul>
+            <h2>NICKNAME:</h2>
+            <ul>English ivy; Common ivy; European ivy</ul>
+            <h2>PLANT TYPE:</h2>
+            <ul>Evergreen climber</ul>
+            <h2>PLANT HEIGHT:</h2>
+            <ul>34.5</ul>
+            <h2>POISENOUS:</h2>
+            <ul>Yes</ul>
+        </div>
+
+        <div class="plants">
+            <img class="snake_plant" src="images/snake-plant.jpg" alt="snake plant also called mother in laws tongues in a dark grey pot">
+            <p>Snake Plant</p>
+            <h2>BOTANICAL NAME:</h2>
+            <ul>Sansevieria trifasciata</ul>
+            <h2>NICKNAME:</h2>
+            <ul>Snake plant; Mother-in-Laws tongue; Vipers bowst</ul>
+            <h2>PLANT TYPE:</h2>
+            <ul>Perennial, indoor</ul>
+            <h2>PLANT HEIGHT:</h2>
+            <ul>64.3 cm</ul>
+            <h2>POISENOUS:</h2>
+            <ul>Yes</ul>
+        </div>
+
+        <div class="plants">
+            <img class="cheese_plant" src="images/cheeseplant.jpg" alt="cheese plant in a traditional plastic pot">
+            <p>Cheese Plant</p>
+            <h2>BOTANICAL NAME:</h2>
+            <ul>Monstera deliciosa</ul>
+            <h2>NICKNAME:</h2>
+            <ul>Cheese plant; Ceriman; Custard plant; Indian ivy; Fruit salad plant; Mexican breadfruit</ul>
+            <h2>PLANT TYPE:</h2>
+            <ul>Evergreen shrub, indoor</ul>
+            <h2>PLANT HEIGHT:</h2>
+            <ul>76.3cm</ul>
+            <h2>POISENOUS:</h2>
+            <ul>yes</ul>
+            </p>
+
+</div>
```

Comment:
strange indentation here. is this div supposed to be part of the code before it?

Reply:
yes, it is the first div at the beginning